navigator/frontend/src/components/Frame.tsx

```diff
@@ -74,7 +74,7 @@ const Frame: React.FC<FrameProps> = ({
           return (
             <Item
               title={title}
-              isActive={false}
+              isActive={activeRoute === Routes.customView && window.location.pathname.split("/")[2] === id}
```

Comment:
I'm far from an expert on React, but do we have the `id` of the current page stored somewhere? Making this based on the current path seems a bit brittle.

Reply:
Custom views are defined as one route and the custom view component renders the content by getting some params whenever a button is clicked. I found no storage of the custom view ID, but it may be stored in a shared context between the  "Frame" and custom view's UI component. Determining the active route for non custom views is also dependent on window.location.pathname:  https://github.com/digital-asset/daml/blob/11e5dd37e730156b97748d67914b5e3f14a48625/navigator/frontend/src/ui-core/src/RouteMatcher/index.ts#L61

Reply:
Fine with me, thanks for clarifying.

Reply:
The id of the custom view you are iterating over is stored in the `id` variable (in scope at this point).

The id of the custom view that you are looking at is stored somewhere in the application state.

Have a look at the  [definition of the page](https://github.com/digital-asset/daml/blob/main/navigator/frontend/src/applets/page/index.tsx#L19) state and try the following:
```
isActive={page.type === 'customview' && page.state.id === id}
```